src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenu.tsx

```diff
@@ -40,7 +40,7 @@ export const FeatureStrategyMenu = ({
                 projectId={projectId}
                 environmentId={environmentId}
                 onClick={onClick}
-                aria-describedby={popoverId}
+                aria-labelledby={popoverId}
```

Comment:
Are there cases where the `popoverId` being undefined can cause problems? Does the popover open as soon as (or before) the item gets focus? I don't know at all, but is it possible that if the popup doesn't open until after the button gets focus, then assistive tech fails to find the label because it's `undefined`?

Reply:
Looks like my Find/Replace went too far. This should stay `aria-describedby` according to the MUI docs.

https://mui.com/material-ui/react-popover/#basic-popover

Not sure how well it manages focus, could be worth looking into.

Reply:
Got it. Is that something you wanna do or would you rather table it for now? ☺️

Reply:
Should proabably be a separete PR at least. I'll make a card.

Reply:
https://trello.com/c/dOUQG85N/722-ensure-that-popovers-focus-handling-works-as-expected